src/components/Chat/PublishModal.tsx

```diff
@@ -153,136 +200,164 @@ export default function PublishModal({
   return (
     <Modal
       portalId="theme-main"
-      containerClassName="inline-block h-[747px] md:min-w-[550px] md:max-w-[1100px] p-6 group/modal"
+      containerClassName={classNames(
+        'group/modal inline-block h-[747px] max-h-full min-w-full max-w-[1100px] !bg-layer-2 md:min-w-[550px]',
```

Comment:
`max-h-full` already in Modal component

```suggestion
        'group/modal inline-block h-[747px] min-w-full max-w-[1100px] !bg-layer-2 md:min-w-[550px]',
```